DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/add-quartus-sln/add_kernel_wrapper.qsys

```diff
@@ -33,14 +33,14 @@ https://fpgasoftware.intel.com/eula.-->
         <ipxact:parameter parameterId="board" type="string">
           <ipxact:name>board</ipxact:name>
           <ipxact:displayName>Board</ipxact:displayName>
-          <ipxact:value>default</ipxact:value>
+          <ipxact:value>Unknown</ipxact:value>
```

Comment:
Is this expected?

Reply:
Yes. don't ask me why it's expected  though; i don't know why it works, just that it compiles with Intel® Quartus® Prime 22.2 and 23.1 :)